projects/go-style-guide/src/app/features/ui-kit/components/accordion-docs/components/accordion-panel-docs/accordion-panel-docs.component.html

```diff
@@ -174,7 +174,6 @@ <h2 class="go-heading-2 go-heading--no-wrap">Delayed Loading</h2>
           <h4 class="go-heading-4 go-heading--underlined">View</h4>
           <go-accordion>
             <go-accordion-panel heading="Not Delayed">
-              <app-loading-test></app-loading-test>
```

Comment:
So removing this is fine, but we probably still want to show _something_ here when this panel is opened. Maybe it can be replaced with a string of text, something like: `This content loaded immediately`